Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.13 as modules
+ADD ./go.mod /m/
+RUN cd /m && go mod download
+FROM golang:1.13 as builder
+
+RUN mkdir -p /opt/resource/
+
+COPY --from=modules /go/pkg/ /go/pkg/
+
+WORKDIR /opt/resource/
+COPY cmd             cmd
+COPY config          config
+COPY kafka           kafka
+COPY filter          filter
+COPY matcher         matcher
+COPY storage         storage
+COPY web             web
+#COPY go.*            ./
```

Comment:
Is it dead code? Let's remove it

Reply:
its for future development. We can add some code to any of this folders and run docker-compose without editing Dockerfile

Reply:
`#COPY go.*            ./` ?

Reply:
removed